charts/rucio-daemons/templates/conveyor-submitter.yaml

```diff
@@ -101,7 +101,7 @@ spec:
+              value: "{{- if .Values.conveyorTransferSubmitter.threads }} --total-threads {{ .Values.conveyorTransferSubmitter.threads }}{{ end }} {{- if .Values.conveyorTransferSubmitter.bulk }} --bulk {{ .Values.conveyorTransferSubmitter.bulk }}{{ end }} {{- if .Values.conveyorTransferSubmitter.groupBulk }} --group-bulk {{ .Values.conveyorTransferSubmitter.groupBulk }}{{ end}}{{- if .Values.conveyorTransferSubmitter.groupPolicy }} --group-policy {{ .Values.conveyorTransferSubmitter.groupPolicy }}{{ end }} {{- if .Values.conveyorTransferSubmitter.mock }} --mock {{ .Values.conveyorTransferSubmitter.mock }}{{ end }} {{- if .Values.conveyorTransferSubmitter.sourceStrategy }} --source-strategy {{ .Values.conveyorTransferSubmitter.sourceStrategy }}{{ end }} {{- if .Values.conveyorTransferSubmitter.excludeRses }} --exclude-rses {{ .Values.conveyorTransferSubmitter.excludeRses }}{{ end}} {{- if .Values.conveyorTransferSubmitter.includeRses }} --include-rses {{ .Values.conveyorTransferSubmitter.includeRses }}{{ end}} {{- if .Values.conveyorTransferSubmitter.rses }} --rses {{ .Values.conveyorTransferSubmitter.rses }}{{ end}} {{- if .Values.conveyorTransferSubmitter.sleepTime }} --sleep-time {{ .Values.conveyorTransferSubmitter.sleepTime }}{{ end}} {{- if .Values.conveyorTransferSubmitter.activities }} --activities {{ .Values.conveyorTransferSubmitter.activities }}{{ end}} {{- if .Values.conveyorTransferSubmitter.excludeActivities }} --exclude-activities {{ .Values.conveyorTransferSubmitter.excludeActivities }}{{ end}} {{- if .Values.conveyorTransferSubmitter.maxSources }} --max-sources {{ .Values.conveyorTransferSubmitter.maxSources }}{{end }} {{- if .Values.conveyorTransferSubmitter.retryOtherFts }} --retry-other-fts {{- end}} {{- if kindIs "float64" .Values.conveyorTransferSubmitter.archiveTimeout }}  --archive-timeout-override {{ .Values.conveyorTransferSubmitter.archiveTimeout }}{{ end}}"
```

Comment:
--retry-other-fts {{- end}}

Optional change: I would remove the "-" in {{- end}} ; it is the only end which strips the white spaces; not problematic now; but could be if somebody adds another option and also strips the white spaces